aries-site/src/examples/templates/dashboard/DashboardExample.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AppIdentity } from 'aries-core';
+import { Box, Button, Header, Heading, Menu } from 'grommet';
+import { DashboardTiles } from '.';
+import { ScreenContainer, SidebarExample } from '..';
+
+export const DashboardExample = ({ mobile }) => {
+  return (
+    <ScreenContainer mobile={mobile}>
+      <Box
+        direction="row"
+        fill
+        width={mobile && { max: 'large' }}
+        height={mobile && { max: 'xlarge' }}
```

Comment:
why are we using `mobile` vs the Responsive Context sizing? Aren't we ignoring the case of tablet?